api/src/controllers/auth.mjs

```diff
@@ -403,18 +403,18 @@ const verifyToken = (token) => {
   const publicKey = utils.getKeys()?.public
 
   return publicKey
-    ?  new Promise((resolve) =>
-      jwt.verify(
-        token,
-        utils.getKeys().public,
-        {
-          audience: 'morio',
-          issuer: 'morio',
-          subject: 'morio',
-        },
-        (err, payload) => resolve(err ? false : payload)
+    ? new Promise((resolve) =>
+        jwt.verify(
+          token,
+          utils.getKeys().public,
+          {
+            audience: 'morio',
+            issuer: 'morio',
+            subject: 'morio',
+          },
+          (err, payload) => resolve(err ? false : payload)
+        )
```

Comment:
This seems to be a whitespace-only change. Let's keep it out of the PR.

Reply:
This change is still here. You can remove it from this PR by running this command in the repo root:
```
git checkout develop ./api/src/controllers/auth.mjs
```

Reply:
Ok